homework27/Test_data.py

```diff
@@ -0,0 +1,55 @@
+"""
+This module provides functions for generating test data
+for use in API requests.
+"""
+
+
+# -*- coding: utf-8 -*-
+import random
+from faker import Faker
+
+fake = Faker()
+
+name = fake.name()
+email = fake.email()
+age = fake.random_int(min=18, max=99)
+
+
+def generate_user_phone_number():
+    """generates phone number in correct format"""
+    msisdn = fake.msisdn()
+    formatted_phone_number = '+' + msisdn
+    return formatted_phone_number
+
+
+phone_number = generate_user_phone_number()
+
+
+def generate_user_address(min_length=10):
+    """generates address in correct format"""
+    generated_address = fake.address()
+    while len(generated_address) < min_length:
+        generated_address = f"{generated_address}, {fake.city()}"
+    return generated_address
+
+
+address = generate_user_address(10)
```

Comment:
do not execute function, import and use where you need it

Reply:
Done